### PR TITLE
Feature/download screenshots

### PR DESCRIPTION
**Pull request by ujappelbe**
I've been using this for all test runs last couple months. Please merge / comment.

Included everyone who might be affected by this since ALL screenshots are now downloaded by default. If you have a lot of screenshots then that is a lot of traffic & data.

### CODE REVIEW

File: testdroid_cmdline.sh [hunk 1]
```diff
@@ -421,8 +462,55 @@ function get_device_result_file {
...
+#########################################
+function get_device_screenshots {
+  test_run_id="$1"
+  device_run_id="$2"
+  device_human_name="$3"
+  test_run_item_url=$(url_from_template "${TD_TEST_RUN_ITEM_URL_TEMPLATE}" "${test_run_id}")
+  device_session_screenshots_url="$test_run_item_url/device-runs/$device_session_id/screenshots"
+  response=$(auth_curl "$device_session_screenshots_url")
+  device_screenshot_ids=$(echo "$response" | jq '.data[] |"\(.id);\(.originalName)"')
+  mkdir -p "${TEST_RESULTS_DIR}/screenshots/${device_human_name}"
```
Comment (antnsi) [inline testdroid_cmdline.sh]: you define screenshots path here and then you define the same path inside function. why not to transfer as argument?

Comment (ujappelbe) [reply testdroid_cmdline.sh]: Good point, I'll revise it.

**Comment by antnsi** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
 	 -i	Set timeout value for project in seconds. Will use 600s (10min) unless specified
 	 -f	Set fail threshold in percentage [0-100], the percentage of test steps that have to pass for the test run to succeed (impacts exit value)
 	 -x	Set device completion threshold in percentage [0-100], the percentage of devices in device group that need to complete for the test run to complete (impacts exit value)
+	 -n	Specify a test_run_id, client will only fetch those results and exit
```
Comment (shusso) [inline README.md]: @ujappelbe  does this `test_run_id` equal the id in `-r     Testdroid test run name`?

Comment (ujappelbe) [reply README.md]: No, this is what they call testRunId and runId in testdroid API.
http://docs.testdroid.com/testdroid-cloud-integration/api/rest-api/

It is true that it is confusing that the id is being referred to here for the first time. Should we include the API-link in the README.md or could there be a better way explaining the id?

Comment (shusso) [reply README.md]: I think this functionality differs compared to the other options as this is something to be done _after_ the run. This was why I was asking. 
Thinking out loud should the `-h` to be divided to two parts:
1. Test-run related functionality
2. Post test-run related functionality

^This is not related purely on this PR though

Comment (ujappelbe) [reply README.md]: Updated usage documentation

**Pull request merged**